src/rawSHA1_ng_fmt_plug.c

```diff
@@ -330,12 +330,10 @@ static void sha1_fmt_set_key(char *key, int index)
 	vtype  Z   = vsetzero();
 	vtype  X   = vloadu(key);
 	vtype  B;
-#if __AVX512F__ || __MIC__
-	uint32_t len = strlen(key);
-#else
-	uint32_t mask = vmovemask_epi8(vcmpeq_epi8(X, Z));
+
+	// FIXME: even uint64_t won't be long enough for AVX-1024
+	uint64_t mask = vcmpeq_epi8_mask(X, Z);
```

Comment:
No, virtual is also slower (from 1408K ro 723K).

Consider this:
```
$ time sleep 1

real	0m1.010s
user	0m0.002s
sys	0m0.003s
```
Here, "real" is wall clock while "user + sys" is how much CPU time you actually used.

Raw shows `crypts / wall clock time` while virtual shows `crypts / system time used` where "system time" is (I think) same as "user + sys" above. So if you measure on a host that is otherwise heavily loaded by other users, the "virtual" will reflect what speed you'd get if the system had not been overloaded.

When using GPU, you often get very high virtual figures because the host CPU was practically not involved. On the other hand when using OpenMP and say 4 threads, you may spend 4 CPU seconds in one "wall clock second" so the virtual figure will be lower than the real.